boris-core/src/Boris/Core/Data.hs

```diff
@@ -286,16 +290,34 @@ data BuildData =
     , buildDataEndTime :: Maybe UTCTime
     , buildDataHeartbeatTime :: Maybe UTCTime
     , buildDataResult :: Maybe BuildResult
-    , buildDataLog :: Maybe LogData
     , buildDataCancelled :: Maybe BuildCancelled
     } deriving (Eq, Ord, Show)
 
 data LogData =
-  LogData {
+    CloudwatchLog CloudwatchLogData
+  | DBLog [DBLogData]
+  deriving (Eq, Ord, Show)
```

Comment:
As above, this looks a bit off.

In english, this reads to me as either _a way to get a log_ or _the actual log itself_. i would of expected it to be just two different ways to get _or_ just the log representation. I can't think of a reason when the client of this type should be deciding what to do in that way.